azure-pipelines.yml

```diff
@@ -183,6 +183,20 @@ stages:
           MGMT_BRANCH: $(BUILD_BRANCH)
           TEST_SET: onboarding_t0
 
+  - job: onboarding_elastictest_t1
+    displayName: "onboarding t1 testcases by Elastictest"
+    timeoutInMinutes: 240
+    continueOnError: true
+    pool: sonic-ubuntu-1c
+    steps:
+      - template: .azure-pipelines/run-test-elastictest-template.yml
+        parameters:
+          TOPOLOGY: t0
```

Comment:
Can you set `STOP_ON_FAILURE: "False"` here? It not, if any script fails in this set, it will block current PR test.

Reply:
Agree, I added it